### PR TITLE
TRIO-49 listing detail page responsiveness

### PR DESCRIPTION
**Pull request by Alex-AK**
Update common component responsiveness for dynamic wrapping.
Move image uploader widget into image component with dynamic rendering.
Add responsiveness to listing detail page.

### CODE REVIEW

File: src/modules/ListingDetailPage/ListingPhotos/index.jsx [hunk 1]
```diff
@@ -3,18 +3,33 @@ import React, { useState } from 'react';
...
+  if (images.length === 0) {
+    return (
+      <Styles>
+        <div className="image-upload-container">
+          <ImageUploader listingId={listingId} columnId={columnId} />
+        </div>
+      </Styles>
+    );
+  }
+
```
Comment (nateboyette) [inline src/modules/ListingDetailPage/ListingPhotos/index.jsx]: We lose the uploader after an image is uploaded, and can't upload anymore photos. 

Also looks like the dispatch/request to upload is failing after clicking done.

**Comment by nateboyette** [comment]
Rebased on master

File: src/modules/ListingDetailPage/ListingPhotos/index.jsx (hunk 1 above)
Comment (Alex-AK) [reply src/modules/ListingDetailPage/ListingPhotos/index.jsx]: Yeah.. didn't think that one through all the way. I'll revisit this (see slack for discussion).

Comment (Alex-AK) [reply src/modules/ListingDetailPage/ListingPhotos/index.jsx]: @nateboyette The image uploader issue on the detail page isn’t something I introduce in this PR. The image uploaded disappears on the master branch too -- same with the failed request. That occurs on master. I propose merging this PR and working on the image uploader separately.

**Pull request merged**